CMSgov/mint-app
# [MINT-2187] Update Operational Solutions

# MINT-2187

## Description

* add `ipcPortal` optional type to `AboutConfigType`
* add conditional `aboutConfig.ipcPortal` render in `SolutionDetails/Solutions/Generic/about.tsx` component
* Add the copy that matches figma
* updates  snaps


## How to test this change

1.  Open the modal for the following solutions
      - RFA
      - 4i
      - ACO-OS 
      - IPC
2. Verify that the new addition matches figma

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
